circuits/types/field/src/lib.rs

```diff
@@ -37,13 +37,13 @@ pub mod to_lower_bits;
 pub mod to_upper_bits;
 pub mod zero;
 
-#[cfg(test)]
-use snarkvm_circuits_environment::assert_scope;
-
 use snarkvm_circuits_environment::prelude::*;
 use snarkvm_circuits_types_boolean::Boolean;
 use snarkvm_utilities::ToBits as TBits;
 
+#[cfg(test)]
+use snarkvm_circuits_environment::assert_scope;
+
```

Comment:
nit: move back please

Reply:
This is used for tests.
Introduced in https://github.com/AleoHQ/snarkVM/commit/ee2a3f98ff422cab5a8ff8e113dd30667356792d

Reply:
nit: move back please